### ISSUE TITLE
Missing documentation for MQTT resource

### ISSUE THREAD

**Issue by atompie** [Tracardi/tracardi#717]
**Is your feature request related to a problem? Please describe.**
Describe how to fill the form . 

![Screenshot from 2022-10-04 21-23-51](https://user-images.githubusercontent.com/16271564/193908294-89848d00-1773-49c4-8fdd-52d97407c84d.png)

Example of the available documentation is in repo https://github.com/Tracardi/tracardi/tracardi-api:

For redis:

```
Resource configuration and set-up

In order to connect to Redis you need the following information:

    url: Url of your redis database
    username: Set-up during redis installation
    password: Set-up during redis installation

All above information is required. None can be left empty.
```

For Mysql:

```
Resource configuration and set-up

Follow the steps:

    Type MySQL HOST - this is the address where the server was installed. Type only IP or the address without http/https, e.g. mysql.server.com
    Type MySQL PORT - most of the time this field does not require any change. Default port is 3306
    Type Username - this is set during the installation of MySQL server
    Type Password - this is set during the installation of MySQL server

```

**Comment by CypherpunkSamurai** [comment]
Hello @atompie,

I would like to contribute to this issue. I had a few questions regarding this:
1. Are we supposed to add the documentation on the form frontend or the docs?

**Comment by atompie** [comment]
@CypherpunkSamurai this is a resource documentation. Of you fill the MD file it will be added to the form automatically.

**Comment by CypherpunkSamurai** [comment]
Ok. I got it. I'll make a draft PR for reviews soon. 👍🏼

**Comment by CypherpunkSamurai** [comment]
Hello @atompie,
I've made a draft pr at #166.

Please Review Changes :)

**Comment by atompie** [comment]
@CypherpunkSamurai  Thanks merged in version 0.7.3.
